README.md

```diff
@@ -1 +1,152 @@
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 hello_rails_back_end <a name="about-project"></a>
+
+**hello_rails_back_end** is a Ruby on Rails API that provides a random greeting to the front end application.
+
+
+## 💻 link to back end <a name="built-with"></a>
+
+
+[Link to Front End](https://github.com/Gh-Reza/hello-react-front-end)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li>Rubocop</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- [x] **Random Greetings**
+- [x] **local API**
+- [x] **Ruby on Rails and React Integration**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Before you begin, make sure you have the following prerequisites installed on your system:
+
+- Ruby: You need Ruby to run the Ruby on Rails application.
+- Bundler: Bundler is used to manage gem dependencies in your Ruby project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+cd my-folder
+git clone https://github.com/Gh-Reza/hello_rails_back_end.git
+```
+
+### Install
+
+Install this project with:
+
+```
+  gem install rails
+  bundle install
+```
+### Usage
+
+To run the project, execute the following command:
+
+```
+rails server
+```
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Reza Merzaie**
+
+- GitHub: [Gh-Reza](https://github.com/Gh-Reza)
+- Twitter: [Reza](https://twitter.com/RezaMerzaie9)
+- LinkedIn: [Reza Merzaie](https://www.linkedin.com/in/reza-merzaie)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Personalized Greetings**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Gh-Reza/hello_rails_back_end/issues).
+
```

Comment:
- [ ] [Optional] I would suggest to fix the broken issues page link. It will make the README file more professional.